spec/models/pending_purchase_spec.rb

```diff
@@ -2,7 +2,11 @@
 
 RSpec.describe PendingPurchase, type: :model do
   let(:user) { create(:user) }
-  let(:pending_purchase) { create(:pending_purchase, user: user, price_at_purchase: 12, quantity: 3) }
+  let(:pending_purchase) { create(:pending_purchase,
+      user: user,
+      price_at_purchase: 12,
+      quantity: 3)
+    }
```

Comment:
`}` at 9, 4 is not aligned with `let(:pending_purchase) { create(:pending_purchase,` at 5, 2.